public/pages/Findings/containers/Findings/Findings.tsx

```diff
@@ -332,6 +520,57 @@ class Findings extends Component<FindingsProps, FindingsState> {
       });
     }
 
+    const tabs = [
+      {
+        id: FindingTabId.DetectionRules,
+        name: (
+          <span>
+            Detection rules (
+            {findingStateByTabId[FindingTabId.DetectionRules].filteredFindings.length})
```

Comment:
`{findingStateByTabId[FindingTabId.DetectionRules]?.filteredFindings.length ?? 0})` to safely access nested properties (filteredFindings and findings) and ?? to provide default values (like 0 or []) if the properties are null or undefined.

Reply:
Not needed since the initial state is not undefined/null but empty arrays